### PR TITLE
Remove constructs and steps related to identity handling

### PR DESCRIPTION
**Pull request by youennf**
Fixes https://github.com/w3c/webrtc-pc/issues/2364

***
<a href="https://pr-preview.s3.amazonaws.com/youennf/webrtc-pc/pull/2390.html" title="Last updated on Dec 6, 2019, 8:08 AM UTC (b9abad9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2390/570f7ff...youennf:b9abad9.html" title="Last updated on Dec 6, 2019, 8:08 AM UTC (b9abad9)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -5523,25 +5413,6 @@ <h2>Methods</h2>
...
-                  "!fetch#concept-cors-check">
-                  CORS cross-origin</a>. These tracks can be supplied to the
-                  <code>addTrack</code> method, and have an
-                  <code><a>RTCRtpSender</a></code> created for them, but
-                  content MUST NOT be transmitted, unless they are also marked
-                  with <code>peerIdentity</code> and they meet the requirements
-                  for sending (see <a>isolated stream</a>).
-                  </p>
-                  <p>All other tracks that are not accessible to the
-                  application MUST NOT be sent to the peer, with silence
```
Comment (jan-ivar) [inline webrtc.html]: It's important not to lose the cross-origin language here.

I suggest a much smaller change s/`being marked with a <code>peerIdentity</code> option or `//

File: webrtc.html [hunk 2]
```diff
@@ -11789,25 +11653,12 @@ <h3><code><dfn>RTCErrorDetailType</dfn></code> Enum</h3>
...
   "sctp-failure",
   "sdp-syntax-error",
   "hardware-encoder-not-available",
   "hardware-encoder-error"
 };</pre>
         </div>
-        <p>
-          The error detail types whose names are prefixed with "idp-" are
-          used by the [[WEBRTC-IDENTITY]] specification. They are described
-          here because a WebIDL enum must be described in one place only.
```
Comment (jan-ivar) [inline webrtc.html]: I suppose the identity spec will need to fake a partial enum here. @henbos am I misremembering or did you run into a similar problem recently? Did you find a good answer?

Comment (jan-ivar) [reply webrtc.html]: I ended up making a `RTCErrorDetailTypeIdp` enum copy.

File: webrtc.html (hunk 1 above)
Comment (henbos) [reply webrtc.html]: +1

File: webrtc.html [hunk 3]
```diff
@@ -11789,25 +11653,12 @@ <h3><code><dfn>RTCErrorDetailType</dfn></code> Enum</h3>
...
   "dtls-failure",
   "fingerprint-failure",
-  "idp-bad-script-failure",
-  "idp-execution-failure",
-  "idp-load-failure",
-  "idp-need-login",
-  "idp-timeout",
-  "idp-tls-failure",
-  "idp-token-expired",
-  "idp-token-invalid",
```
Comment (alvestrand) [inline webrtc.html]: We still don't have "partial enum" in WebIDL. How do we handle the extension?

**Comment by alvestrand** [comment]
Decision: Leave the language about cross-origin tracks in for now, to assist in discussing #2397

File: webrtc.html (hunk 1 above)
Comment (youennf) [reply webrtc.html]: I left some of the text. I still think this is unnecessary and not well specified but we can improve this as a follow-up

**Comment by youennf** [comment]
Sure, I believe I don't have write access so go ahead!

**Pull request merged**